ui/src/components/pamFileList/index.vue

```diff
@@ -439,7 +439,7 @@ const handleBeforeLeave = (tabName: string) => {
 const columns = createColumns();
 
 onMounted(() => {
-  useFileManage();
+  // useFileManage();
   mittBus.on('open-fileList', handleOpenFileList);
   mittBus.on('open-setting', handleOpenSetting);
 });
```

Comment:
There does not appear to be any code provided in your message that needs checking or analyzing. Please provide a snippet of code for me to assist with any irregularities, potential issues, or optimizations you may have encountered.